### PR TITLE
idid: init at 2.1.0

### PR DESCRIPTION
**Pull request by dbalan**
Adds a new package idid to nixpkgs.

###### Motivation for this change
Adds a new package that I wrote and use (it was a script in my personal nix config so far)

###### Things done

- [x] Tested using sandboxing ([nix.useSandbox](http://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](http://nixos.org/nix/manual/#sec-conf-file) on non-NixOS)
- Built on platform(s)
   - [x] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [x] Tested compilation of all pkgs that depend on this change using `nix-shell -p nix-review --run "nix-review wip"`
- [x] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [x] Ensured that relevant documentation is up to date
- [x] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

###### Notify maintainers

cc @

### CODE REVIEW

File: maintainers/maintainer-list.nix [hunk 1]
```diff
@@ -1492,6 +1492,12 @@
     githubId = 49904992;
     name = "Dawid Sowa";
   };
+  dbalan = {
```
Comment (mmahut) [inline maintainers/maintainer-list.nix]: Please do you mind splitting this change into its separate commit?

Comment (dbalan) [reply maintainers/maintainer-list.nix]: @mmahut Thanks for taking a look, I've split the change into two commits.

File: pkgs/applications/office/idid/default.nix [hunk 2]
```diff
@@ -0,0 +1,37 @@
+{ mkDerivation, haskellPackages, fetchFromGitHub, lib}:
+
+let
+  version = "2.1.0";
+in
+
+mkDerivation {
+  pname = "idid";
+  inherit version;
```
Comment (Lassulus) [inline pkgs/applications/office/idid/default.nix]: you can write it like this:
```
mkDerivation rec {
  pname = "idid";
  version = "2.1.0";
```

**Comment by Lassulus** [comment]
@ajs124 told me you should package it on hackage ;)

**Comment by dbalan** [comment]
@Lassulus Thanks! I'll try to include it via hackage.

**Pull request closed without merging**